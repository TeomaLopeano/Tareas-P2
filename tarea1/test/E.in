insertarEnColCadenas 5 5.0 4
cantidadColCadenas 1
cantidadColCadenas 5
estaEnColCadenas 3 0
estaEnColCadenas 5 4
infoEnColCadenas 5 4
insertarEnColCadenas 3 13.1 4
insertarEnColCadenas 3 13.1 5
estaEnColCadenas 3 0
estaEnColCadenas 3 4
estaEnColCadenas 3 5
insertarEnColCadenas 5 15.0 5
insertarEnColCadenas 6 6.0 5
insertarEnColCadenas 7 7.0 5
cantidadColCadenas 1
cantidadColCadenas 4
cantidadColCadenas 5
infoEnColCadenas 5 4
infoEnColCadenas 5 5
Fin