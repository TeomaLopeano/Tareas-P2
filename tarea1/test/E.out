1>Se insertó (5,5.00) en la cadena 4.
2>La cantidad en la cadena 1 es 0.
3>La cantidad en la cadena 5 es 0.
4>3 no está en la cadena 0.
5>5 está en la cadena 4.
6>La primera aparición de 5 en la cadena 4 es (5,5.00)
7>Se insertó (3,13.10) en la cadena 4.
8>Se insertó (3,13.10) en la cadena 5.
9>3 no está en la cadena 0.
10>3 está en la cadena 4.
11>3 está en la cadena 5.
12>Se insertó (5,15.00) en la cadena 5.
13>Se insertó (6,6.00) en la cadena 5.
14>Se insertó (7,7.00) en la cadena 5.
15>La cantidad en la cadena 1 es 0.
16>La cantidad en la cadena 4 es 2.
17>La cantidad en la cadena 5 es 4.
18>La primera aparición de 5 en la cadena 4 es (5,5.00)
19>La primera aparición de 5 en la cadena 5 es (5,15.00)
20>Fin.
